static/sass/_lagunita.scss

```diff
@@ -287,6 +287,11 @@ $video-thumb-url: '../themes/lagunita/images/stanford-online-video-thumb.png';
     }
 }
 
+.course-info header.course-profile {
```

Comment:
Discussed with @caesar2164 it's safer to keep the ```!important``` since upstream could change selector rules that cause it to get overridden.